demo/app-root.ts

```diff
@@ -46,6 +57,26 @@ export class AppRoot extends LitElement {
   @query('#include-selected-option')
   private includeSelectedOptionCheck!: HTMLInputElement;
 
+  constructor() {
+    super();
+    // Copy sample userlist data
+    this.userlistData = userlistTestData.map(item => item);
```

Comment:
Possible alternative to consider for this shallow array clone:

```suggestion
    this.userlistData = [...userlistTestData];
```

Reply:
Great suggestion, less familiar to me, but seems better style conformity.